packages/server/postgres/queries/getUsersByEmail.ts

```diff
@@ -0,0 +1,22 @@
+import {getUsersByEmailQuery} from './generated/getUsersByEmailQuery'
+import getPg from '../getPg'
+import catchAndLog from '../utils/catchAndLog'
+import AuthIdentity from '../../database/types/AuthIdentity'
+import User from '../../database/types/User'
+import {AuthTokenRole} from 'parabol-client/types/constEnums'
+
+export const getUsersByEmail = async (emails: string[]): Promise<User[]> => {
```

Comment:
NIT: as we're passing `emails` plural into this function, perhaps `getUsersByEmails` would be more descriptive?